lib/config.js

```diff
@@ -18,6 +18,8 @@ const startsWith = require('lodash/fp/startsWith')
 const fs = require('fs')
 const ServerError = require('../errors/server-error')
 
+const SECRET_NAMES = {} // { prefix ⇒ { name ⇒ true } }
```

Comment:
It should only be uppercase if its a constant. Now that we're modifying it, it should just be `secretNames`.